.pipelines/templates/template-clean-subscription.yml

```diff
@@ -0,0 +1,44 @@
+parameters:
+- name: purgeTTL
+  displayName: Timeout before resources are force cleaned.
+  type: string
+  default: 6h
+- name: subscriptionCredentials
+  displayName: Subscription properties required to login
+  type: string
+  default: ''
+- name: subscriptionId
+  displayName: SubscriptionId to clean
+  type: string
+  default: ''
+- name: workingDirectory
+  type: string
+  default: ''
+
+jobs:
+- job: Clean_E2E
+  pool:
+    vmImage: ubuntu-latest
+
+  steps:
+  - template: ./template-setup-golang-env.yml
+    parameters:
+      gobin: ${{ variables.GOBIN }}
+      gopath: ${{ variables.GOPATH }}
+      goroot: ${{ variables.GOROOT }}
+      modulePath: ${{ variables.modulePath }}
+
+  - script: |
+      set -e
+
+      echo ${{ parameters.subscriptionCredentials }} | base64 -d -w 0 > spn.json
+      export AZURE_CLIENT_ID=$(cat spn.json | jq -r '.clientId')
+      export AZURE_CLIENT_SECRET=$(cat spn.json | jq -r '.clientSecret')
+      export AZURE_TENANT_ID=$(cat spn.json | jq -r '.tenantId')
+      export AZURE_SUBSCRIPTION_ID=$(parameters.subscriptionId)
+      export AZURE_PURGE_TTL="${{ parameters.purgeTTL }}""
+      rm spn.json
+
+      go run ./hack/clean --dryRun
```

Comment:
this is going to stop cleaning in MSFT's subscription for now? would rather not.